css/style.css

```diff
@@ -1,112 +1,110 @@
 * {
-    margin: 0;
-    padding: 0;    
-    box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
 body {
-    background-color: gray;
+  background-color: gray;
 }
 
 #topArea {
-    background: url(/images/topAreaBackground.jpg);
-    /* background-repeat: no-repeat; */
-    background-size: 100% 100%;        
-    height: 100vh;
+  background: url(/images/topAreaBackground.jpg);
+  background-size: 100% 100%;
+  height: 100vh;
 }
 
 header {
-    display: flex;
-    position: fixed;
-    width: 100%;
-    justify-content: space-between;    
-    padding-bottom: 80px;
-    margin-bottom: 80px;
+  display: flex;
+  position: fixed;
+  width: 100%;
+  justify-content: space-between;
+  padding-bottom: 80px;
+  margin-bottom: 80px;
 }
 
 img {
-    height: 80px;
-    width: 450px;        
+  height: 80px;
+  width: 450px;
 }
 
-#nav-bar { 
-    display: flex;
-    align-items: center;
-    justify-content: center;      
-    top: 0;    
-    width: 40%;
-    text-align: center;
+#nav-bar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  width: 40%;
+  text-align: center;
 }
 
-.nav-link {    
-    margin-left: 25px;
-    font-size: 2.75rem;
-    font-family: helvetica;
+.nav-link {
+  margin-left: 25px;
+  font-size: 2.75rem;
+  font-family: helvetica, sans-serif;
 }
 
 #pageContainer1 {
```

Comment:
[Optional] 

- [ ]  You can use dashes '-' instead of camelcase for IDs.

Reply:
[Optional] - [ ]  You can use dashes '-' instead of camelcase for IDs.  This will keep the style of your code consistent with your classes and your IDs and make it easier for you to remember! 👍